src/electron/routing_service.ts

```diff
@@ -234,4 +217,22 @@ export class RoutingDaemon {
   public set onNetworkChange(newListener: ((status: TunnelStatus) => void) | undefined) {
     this.networkChangeListener = newListener;
   }
+
+  public static installRoutingServices(): Promise<boolean> {
```

Comment:
Make this a standalone function to make it clearer that there's no need to access private members of `RoutingDemon`